### ISSUE TITLE
Dot11Beacon network_stats bugs with Python3

### ISSUE THREAD

**Issue by 1earch** [secdev/scapy#1872]
Getting network statistics from Dot11Beacon packets thanks to `network_stats` with Python3 raises an error.

#### Environment

- Scapy version: 2.4.2
- Python version: 3.7.2
- Operating system : Arch Linux

#### How to reproduce

Put your wireless card in monitor mode then listen for beacon frames:

```python
from scapy.all import *
import six


def filter_beacon(p):
    """ Filter WiFi beacon and probe response packets. """
    return p.haslayer(Dot11Beacon)


def print_ap(p):
    """ Print access point. """
    netstats = p[Dot11Beacon].network_stats()

    # Get AP details
    bssid    = p.addr3
    ssid     = netstats['ssid']
    channel  = netstats['channel']
    crypto   = '/'.join(netstats['crypto'])

    # Print discovered AP
    print(ssid + " (" + bssid + ") on channel " + str(channel) + " with crypto " + crypto)


""" Refresh APs list. """
sniff(iface="mon0", lfilter=filter_beacon, prn=print_ap)
```
#### Actual result with Python2

```
Livebox-1B23 (a4:3e:51:0b:1b:23) on channel 6 with crypto WPA2/WPA
Livebox-1B23 (a4:3e:51:0b:1b:23) on channel 6 with crypto WPA2/WPA
...
```

#### Actual result with Python3

```
Traceback (most recent call last):
  File "min_script.py", line 25, in <module>
    sniff(iface="mon0", lfilter=filter_beacon, prn=print_ap)
  File "/usr/lib/python3.7/site-packages/scapy/sendrecv.py", line 886, in sniff
    r = prn(p)
  File "min_script.py", line 12, in print_ap
    netstats = p[Dot11Beacon].network_stats()
  File "/usr/lib/python3.7/site-packages/scapy/layers/dot11.py", line 430, in network_stats
    p.info.startswith('\x00P\xf2\x01\x01\x00'):
TypeError: startswith first arg must be bytes or a tuple of bytes, not str
```

#### Expected result

As in Python2

**Comment by MisterBianco** [comment]
Sorry to resurrect a hopefully dead thread, but I am getting this issue, will this be fixed in 2.4.3?
and if so when should that drop? :)

**Comment by gpotter2** [comment]
Hi !
This is already fixed on master, which will be used for 2.4.3. I'm not in charge of the scheduling.

@guedou Do you think we could merge the few remaining bug fixes, pack 2.4.3rc2 soon, and work our way to the final release (with a proper freeze) ? Thanks !

**Comment by MisterBianco** [comment]
I am actually in the process of a major rewrite that will change alot of things including the use of notdecoded :) I was originally only using it out of necessity. I am excited for the network_Stats addition and will be very excited to make boopsuite even better :)
